pkg/volume/csi/csi_mounter.go

```diff
@@ -87,8 +87,12 @@ func getTargetPath(uid types.UID, specVolumeID string, host volume.VolumeHost) s
 var _ volume.Mounter = &csiMountMgr{}
 
 func (c *csiMountMgr) CanMount() error {
-	//TODO (vladimirvivien) use this method to probe controller using CSI.NodeProbe() call
-	// to ensure Node service is ready in the CSI plugin
+	ctx, cancel := grpctx.WithTimeout(grpctx.Background(), csiTimeout)
+	defer cancel()
+	if err := c.csiClient.NodeProbe(ctx, csiVersion); err != nil {
+		glog.Error(log("mounter.CanMount failed to probe driver: %v", err))
```

Comment:
Do we need to enhance above error message as `mounter.CanMount failed to probe CSI driver`  for better clarity?

Reply:
ACK.